src/install-elasticsearch.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Are these scripts being used?

Btw, I've just noticed that we have a `src` and a `scripts` directory, with no apparent difference between the two. Could we just move these scripts into `scripts` and delete this `src` directory?

Reply:
> Are these scripts being used??

Not at the moment, but we'd need them when we automate booting up Elastic and Kibana servers alongside the genesis node.

>  Could we just move these scripts into scripts and delete this src directory?
Totally, yea. I can do that in this PR 👍